.dockerignore

```diff
@@ -4,8 +4,11 @@ deploy/
 .circleci/
 .git/
 .github/
-.vs/
+**/.vs/
 .vscode/
```

Comment:
While you're in here, do you mind?
```suggestion
**/.vscode/
```

Reply:
Addressed in 60ad318fb3fbe2d48b3477163178326edb021cc3